src/fragments/lib/push-notifications/android/getting_started/20_cli_resources.mdx

```diff
@@ -0,0 +1,39 @@
+> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
+
+<Callout warning>
+
+Push Notifications requires version **10.8.0+** of the Amplify CLI. You can check your current version by typing `amplify -version` and upgrade to the latest version by typing `amplify upgrade`.
+
+</Callout>
+
+<Callout>
+
+If you want to use existing AWS Pinpoint resources [follow this guide](/lib/push-notifications/existing-resources) instead.
+
+</Callout>
+
+To start provisioning push notifications resources in the backend, go to your project directory and execute the command:
+
+```sh
+amplify add notifications
+```
+
+Choose *FCM* when promoted:
+
+```console
+? Choose the push notification channel to enable.
+APNS
+> FCM
+Email
+SMS
+```
+
+The CLI will prompt for your *ApiKey*, paste the **Token** you copied from the Firebase console.
+
+Once all your configuration is complete run the following:
+
+```sh
+amplify push
+```
+
+Upon completion, `amplifyconfiguration.json` will be updated to reference the newly provisioned backend push notifications resources. Note that this file should already be generated for you by the Amplify CLI as a part of your project if you followed the [project setup walkthrough](/lib/project-setup/prereq).
```

Comment:
This link to CLI setup is different from https://github.com/aws-amplify/docs/pull/5242/files#diff-bef234b6b7c1d11c6c485db1e1f31304b9011716c85479b805b031cb638ae4beR2 above. This can be confusing as these point to different products of Amplify.

Reply:
One points to CLI -> CLI setup and other points to Android -> CLI setup.